### DISCUSSION TITLE
Improving Mac experience

### DISCUSSION THREAD

**Discussion by Overbuffer** [jeffvli/sonixd#280] [Ideas]
Hi there!

Congratulations, Sonixd is a great piece of software!  Because of that I would love to get in all my machines.

Here are some ideas to improve Sonixd on Mac OS systems:
1. **App installation / Upgrade vía brew**:
As many other projects I really think for usability and discoverability the app should be listed in brew Cask on https://formulae.brew.sh/cask/.
BTW: I don't know why but I could not get to work the update in Mac OS... However I would be not necessary if we could use 'brew' :)
2. **Auto switch dark/light theme with system**:
If I'm not mistaken, the currents versions of electron supports this:
https://www.electronjs.org/docs/latest/tutorial/dark-mode#macos-settings
 I don't know how much effort we will need to integrate this but I will be amazing

Thanks & Keep up the nice work! 

BTW: I could offer some support (at least with TestFlight ) for this task. Let me know if I can be of help for you ;)

**Comment by jeffvli** [comment]
I'll look into seeing if I can get it onto Homebrew. In regards to the autoupdates, I think it's broken due to missing the [code signing](https://www.electron.build/code-signing) requirement. Don't remember if it's because I needed to join the Apple Developer program or not, but I'll double check that again.

The dark/light theme switcher is interesting, though not sure how much demand there is for that feature. I'd say that most users are in a set and forget mindset regarding themes, but I'll think about adding a `Use system theme` option that will autoselect either light or dark.

**Reply by Overbuffer** [reply]
Nice! Let me know if I could be of any help.

About the auto-system-theme IMHO is important because, if I remember correctly, "automatic"is the default choice during Mac OS user setup since several versions.
